explorer/src/components/instruction/system/SystemDetailsCard.tsx

```diff
@@ -39,6 +39,8 @@ type DetailsProps = {
   ix: ParsedInstruction;
   result: SignatureResult;
   index: number;
+  innerCards?: JSX.Element[];
```

Comment:
I imagine some of the instruction cards would never have inner instructions. I kept them in for consistency, but am fine with removing.